src/app/blog/author/[slug]/page.tsx

```
@@ -0,0 +1,182 @@
+        <div className="relative container-wide text-center">
+          <div className="inline-flex flex-center gap-2 px-4 py-2 mb-8 rounded-full border border-cyan-300 bg-cyan-400/10 text-cyan-400 font-semibold text-lg">
+            <UserIcon className="w-5 h-5" />
+            Author
+          </div>
+
+          {author.profile_image && (
+            <div className="flex justify-center mb-6">
+              {/* eslint-disable-next-line @next/next/no-img-element */}
+              <img
+                src={author.profile_image}
+                alt={`${author.name}'s profile picture`}
```

Biome (2.1.2)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)